docs/client/README.md

```diff
@@ -32,6 +32,12 @@ After you have installed `kubectl` or `kn`, these tools will search for the `kub
 
 A `kubeconfig` file is usually automatically created when you create a Kubernetes cluster.
 
+You can also specify a config file in the following ways:
+
+- Setting the environment variable `$KUBECONFIG`, and point it to the kubeconfig file.
+
+- Using the `kn` CLI `--config` option, for example, `kn --config path/to/config service list`. The default config is at `~/.config/kn/config.yaml`.
```

Comment:
```suggestion
- Using the `kn` CLI `--config` option, for example, `kn service list --config path/to/config.yaml`. The default config is at `~/.config/kn/config.yaml`.
```

Reply:
Hi thanks for applying my feedback! Sorry, just one more suggestion. I've usually seen the option at the end of the command and also it would be nice to have a file extension at the end of the path.

Reply:
okay, looks good. I'll commit the suggestion :)